cmd/cli/kubectl-kyverno/commands/apply/command_test.go

```diff
@@ -823,3 +848,69 @@ func TestCommandHelp(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, strings.HasPrefix(string(out), cmd.Long))
 }
+func Test_Apply_MutatingAdmissionPolicies(t *testing.T) {
+	testcases := []*TestCase{
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"../../../../../test/cli/test-mutating-admission-policy/policy.yaml"},
+				ResourcePaths: []string{"../../../../../test/cli/test-mutating-admission-policy/resource.yaml"},
+				PolicyReport:  true,
+				Variables:     []string{"request.operation=CREATE"},
+			},
+			expectedPolicyReports: []policyreportv1alpha2.PolicyReport{{
+				Summary: policyreportv1alpha2.PolicyReportSummary{
+					Pass:  1,
+					Fail:  0,
+					Skip:  0,
+					Error: 0,
+					Warn:  0,
+				},
+			}},
+		},
+	}
+
+	for _, tc := range testcases {
+		t.Run("MAP test", func(t *testing.T) {
+			verifyTestcase(t, tc, compareSummary)
+		})
+	}
+}
+
+func Test_LoadMAPFile(t *testing.T) {
+	loaderResults, err := policy.Load(nil, "", "../../../../../test/cli/test-mutating-admission-policy/policy.yaml")
+	if err != nil {
+		t.Fatalf("Error loading policy file: %v", err)
+	}
+	if loaderResults == nil || len(loaderResults.MAPs) == 0 {
+		t.Fatalf("Expected at least one MAP, but got 0")
+	}
+	t.Logf("Loaded MAPs: %+v", loaderResults.MAPs)
+}
+
+func Test_Apply_MutatingAdmissionPolicy_WithBinding(t *testing.T) {
+	testcases := []*TestCase{
+		{
+			config: ApplyCommandConfig{
+				PolicyPaths:   []string{"../../../../../test/cli/test-mutating-admission-policy-binding/policy.yaml"},
+				ResourcePaths: []string{"../../../../../test/cli/test-mutating-admission-policy-binding/resource.yaml"},
+				PolicyReport:  true,
+				Variables:     []string{"request.operation=CREATE"},
+			},
+			expectedPolicyReports: []policyreportv1alpha2.PolicyReport{{
+				Summary: policyreportv1alpha2.PolicyReportSummary{
+					Pass:  1,
+					Fail:  0,
+					Skip:  0,
+					Error: 0,
+					Warn:  0,
+				},
+			}},
+		},
+	}
+
+	for _, tc := range testcases {
+		t.Run("MAP with MAPB test", func(t *testing.T) {
+			verifyTestcase(t, tc, compareSummary)
+		})
+	}
+}
```

Comment:
Instead of creating a new function for the MAP binding, you can move it to `Test_Apply_MutatingAdmissionPolicies`. It should have all the test cases for MAPs.